ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastDownloaderFactory.java

```diff
@@ -57,12 +57,13 @@
 
     final Path fastSyncDataDirectory = dataDirectory.resolve(FAST_SYNC_FOLDER);
     final FastSyncStateStorage fastSyncStateStorage =
-        new FastSyncStateStorage(fastSyncDataDirectory);
+        new FastSyncStateStorage(fastSyncDataDirectory, protocolSchedule);
 
     if (syncConfig.getSyncMode() != SyncMode.FAST) {
       if (fastSyncStateStorage.isFastSyncInProgress()) {
         throw new IllegalStateException(
-            "Unable to change the sync mode when fast sync is incomplete, please restart with fast sync mode");
+            "Unable to change the sync mode when fast sync is incomplete, please restart with fast"
```

Comment:
Is it necessary to split these strings into 120 column bits?  It increases the line count in an already large PR.

Reply:
This is an artifact of me running google-java-format externally to intellij sometimes as a git hook, apparently I don't have it set  up with the same settings as intellij. I'll remove them.